### PR TITLE
toast and grid html

### PR DESCRIPTION
**Pull request by b0llu**
Added Toast and Grid to Documentation site
image
image
image
image
image
image

### CODE REVIEW

File: Components/toast/toast.html [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+                        <li><a class="link-hover" href="../Card/card.html">Card</a></li>
+                        <li><a class="link-hover" href="../Image/image.html">Image</a></li>
+                        <li><a class="link-hover" href="../Input/input.html">Input</a></li>
+                        <li><a class="link-hover" href="./text-utilites.html">Text Utilities</a></li>
+                        <li><a class="link-hover" href="../lists/lists.html">Lists</a></li>
+                        <li><a class="link-hover" href="../navigation/navigation.html">Navigation</a></li>
+                        <li><a class="link-hover" href="../modal/modal.html">Modal</a></li>
+                        <li><a class="link-hover" href="../rating/rating.html">Rating</a></li>
+                        <li><a class="link-active" href="./toast.html">Toast</a></li>
+                        <li><a class="link-hover" href="../grid/grid.html">Grid System</a></li>
```
Comment (anik31) [inline Components/toast/toast.html:80]: Maintain consistency here, some list items are in one line & some are not.

Comment (b0llu) [reply Components/toast/toast.html:80]: thats a bug of prettier. I have tried to fix it but when i save it changes back to this layout cant really do much but thanks for pointing that out!

File: Components/grid/grid.html [hunk 2]
```diff
@@ -0,0 +1,161 @@
...
+            <p class="sub-content">
+                Use our powerful mobile-first flexbox grid to build layouts of all shapes and sizes thanks to a twelve
+                column system, six default responsive tiers, Sass variables and mixins, and dozens of predefined
+                classes.
+            </p>
+            <h1 class="content-title">Grid- 2</h1>
+            <div class="content-container">
+                <div class="grid-2">
+                    <div class="box">1</div>
+                    <div class="box">2</div>
```
Comment (manojsarna) [inline Components/grid/grid.html:97]: Lokks Great!!

File: Components/toast/toast.html [hunk 3]
```diff
@@ -0,0 +1,127 @@
...
+                <div class="toast-success">
+                    Your product has been successfully added to the card.
+                </div>
+            </div>
+            <iframe class="iframe-h-25"
+                src="https://carbon.now.sh/embed?bg=rgba%28171%2C+184%2C+195%2C+1%29&t=lucario&wt=none&l=htmlmixed&width=777&ds=true&dsyoff=20px&dsblur=68px&wc=true&wa=true&pv=56px&ph=56px&ln=false&fl=1&fm=Hack&fs=14px&lh=133%25&si=false&es=2x&wm=false&code=%253Cdiv%2520class%253D%2522toast-success%2522%253E%250A%2509Your%2520product%2520has%2520been%2520successfully%2520added%2520to%2520the%2520card.%250A%253C%252Fdiv%253E"
+                sandbox="allow-scripts allow-same-origin">
+            </iframe>
+            <div class="divider"></div>
+            <h1 class="content-title">Error Toast</h1>
```
Comment (manojsarna) [inline Components/toast/toast.html:105]: can use alternative to iframes , you can use Prism.js to reduce your network calls for the code the snippets.

**Review by manojsarna — APPROVED** [review]
Overall Looks Good!!

You Can merge!!

File: Components/grid/grid.html (hunk 2 above)
Comment (b0llu) [reply Components/grid/grid.html:97]: thanks

File: Components/toast/toast.html (hunk 3 above)
Comment (b0llu) [reply Components/toast/toast.html:105]: sure next time i will keep that in mind

**Pull request merged**